src/services/LbpProjectTokenPriceService.ts

```diff
@@ -0,0 +1,144 @@
+import { autoinject } from "aurelia-framework";
+import * as moment from "moment-timezone";
+import Moment = moment.Moment;
+
+@autoinject
+export class LbpProjectTokenPriceService {
+
+  maxSupply: number;
+  projectTokenInitialLbpAmount: number;
+  fundingTokenInitialLbpAmount: number;
+  startWeightProjectToken: number;
+  endWeightProjectToken: number;
+  startDate: string;
+  endDate: string;
+  roundedStartDate: Moment;
+  roundedEndDate: Moment;
+  lbpDurationInHours: number;
+  pricePerFundingToken: number;
+
+
+  constructor(
+    maxSupply: number,
```

Comment:
you can specify these constructor params as private and they will be automatically initialized to the instance variables of the same name.